### PR TITLE
[Hyperengage Actions] - Added Documentation for New Destination

### PR DESCRIPTION
**Pull request by zhadier39**
### Proposed changes

Added documentation for the [Hyperengage Actions Destination.](https://github.com/segmentio/action-destinations/pull/1621)

### Merge timing
- ASAP

### CODE REVIEW

**Review by joe-ayoub-segment — CHANGES_REQUESTED** [review]
Note for docs team reviewer: 

This Integration has 3 Actions, one to handle each of track(), identify() and group calls.

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+<!-- Include a brief description of the destination here, along with a link to your website. -->
+<!-- In the section below, explain the value of this actions-based destination. If you don't have a classic version of the destination, remove this section. -->
+[Hyperengage](https://hyperengage.io/){:target="_blank"} tracks thousands of data points to trigger smart alerts on hidden opportunities when the accounts are ready for upsell, or likely to churn. By integrating product data into your GTM strategy, our platform empowers CSM’s and AE’s to achieve up to 5x higher lead conversion and better retention and adoption.
+
+## Benefits of Hyperengage (Actions)
+
+Hyperengage (Actions) offers several advantages:
+
+- **Seamless Data Mapping.** Hyperengage streamlines the process of syncing your user and account data. By linking your account, identify, track, and group calls with user or account data will be processed by Hyperengage, bypassing the need for manual API integrations.
```
[the remark this answers is not in the document]
Comment (zhadier39) [reply src/connections/destinations/catalog/hyperengage/index.md]: Hey @joe-ayoub-segment ,
Thank you for your feedback, I've rephrased the sentence to sound less ambigious.

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+<!-- This template is meant for Actions-based destinations that do not have an existing Classic or non-Actions-based version. For Actions Destinations that are a new version of a classic destination, see the doc-template-update.md template. -->
+
+{% include content/plan-grid.md name="actions" %}
+
+<!-- Include a brief description of the destination here, along with a link to your website. -->
+<!-- In the section below, explain the value of this actions-based destination. If you don't have a classic version of the destination, remove this section. -->
+[Hyperengage](https://hyperengage.io/){:target="_blank"} tracks thousands of data points to trigger smart alerts on hidden opportunities when the accounts are ready for upsell, or likely to churn. By integrating product data into your GTM strategy, our platform empowers CSM’s and AE’s to achieve up to 5x higher lead conversion and better retention and adoption.
+
+## Benefits of Hyperengage (Actions)
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:16]: hi @tcgilbert - do we typically include this section if there is not a classic destination already available to folks?

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+# Hyperengage Destination
+title: Hyperengage Destination
+hide-boilerplate: true
+hide-dossier: true
+---
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:6]: This destination is in the catalog as Hyperengage (Actions), so I've updated the destination's name accordingly. I also added the private and beta tags to denote the current status of the destination. The destination ID is pulled from Segment's Public API - including it in the frontmatter like this allows us to programmatically build content on the site (for example, this destination will appear on the [list of Actions destinations](https://segment.com/docs/connections/destinations/actions/#destination-actions-compatibility)).

```suggestion
---
title: Hyperengage (Actions) Destination
hide-boilerplate: true
hide-dossier: true
private: true
beta: true
id: 651c1db19de92d8e595ff55d
---
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+<!-- Include a brief description of the destination here, along with a link to your website. -->
+<!-- In the section below, explain the value of this actions-based destination. If you don't have a classic version of the destination, remove this section. -->
+[Hyperengage](https://hyperengage.io/){:target="_blank"} tracks thousands of data points to trigger smart alerts on hidden opportunities when the accounts are ready for upsell, or likely to churn. By integrating product data into your GTM strategy, our platform empowers CSM’s and AE’s to achieve up to 5x higher lead conversion and better retention and adoption.
+
+## Benefits of Hyperengage (Actions)
+
+Hyperengage (Actions) offers several advantages:
+
+- **Seamless Data Mapping.** Hyperengage streamlines the process of syncing your user and account data. When you link your data, Hyperengage will automatically process Segment's identify, track, and group calls, eliminating the necessity for manual API integrations."
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:20]:
```suggestion
- **Seamless Data Mapping**: Hyperengage streamlines the process of syncing your user and account data. When you link your data with Segment, Hyperengage automatically processes Segment's Identify, Track, and Group calls, eliminating the need for manual API integrations.
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+- **Pre-configured Mapping.** We have prebuilt mappings tailored for Hyperengage with all the necessary parameters, reducing the need for manual customization.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:21]:
```suggestion
- **Pre-configured Mappings**: The Hyperengage (Actions) Destination has prebuilt mappings tailored for Hyperengage with all the necessary parameters, reducing the need for manual customization.
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+- **Direct Data Transfer.** Data moves straight from Segment to Hyperengage, eliminating the need for third-party intermediaries.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:22]:
```suggestion
- **Direct Data Transfer**: Data moves straight from Segment to Hyperengage, eliminating the need for third-party intermediaries.
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+- **Event Tracking and User Identification.** With Actions-based destinations, you can keep track of events and identify users and organizations in Hyperengage.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:23]:
```suggestion
- **Event Tracking and User Identification**: With Segment's Actions-based destination, you can keep track of events and identify users and organizations in Hyperengage.
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+<!-- The section below describes how to activate and set up the destination. Include any additional configuration steps not mentioned. For example, acquiring an API key from your system and any setup steps needed to establish a connection to the destination. -->
+
+## Getting Started
+
+1. Navigate to the Segment web app and select **Catalog**, then pick **Destinations**.
+2. In the Destinations Catalog, search for "hyperengage" and choose the destination.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:30]:
```suggestion
2. In the Destinations Catalog, search for "Hyperengage" and select the Hyperengage (Actions) destination.
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+3. Opt for **Configure Hyperengage**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:31]:
```suggestion
3. Click **Add destination**.
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+4. Choose an existing Source to connect to Hyperengage (Actions).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:32]:
```suggestion
4. Choose an existing Source to connect to Hyperengage (Actions) and click **Next**.
5. Enter a name for your destination and click **Create destination**.
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+5. Access the [Hyperengage App](https://hyperengage.io/){:target="_blank"}, proceed to **Integration Settings**, and copy the **API Key** & **Workspace Identifier**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:33]:
```suggestion
6. Open the [Hyperengage App](https://hyperengage.io/){:target="_blank"}, proceed to **Integration Settings**, and copy the API Key and Workspace Identifier.
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+6. Paste the "API Key" & "Workspace Identifier" into the "Hyperengage (Actions)" destination settings within Segment.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:34]:
```suggestion
7. Open the Segment app, navigate to your Hyperengage (Actions) destination, and paste the API Key and Workspace Identifier into the destination's settings page.
```

**Review by forstisabella — CHANGES_REQUESTED** [review]
a couple of changes and we'll be good to go 🚀

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
+---
+# Hyperengage Destination
+title: Hyperengage Destination
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:3]:
```suggestion
title: Hyperengage (Actions) Destination
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+hide-boilerplate: true
+hide-dossier: true
```
Comment (forstisabella) [inline src/connections/destinations/catalog/hyperengage/index.md:5]:
```suggestion
hide-dossier: true
beta: true
private: true
id: 651c1db19de92d8e595ff55d
```

**Review by forstisabella — APPROVED** [review]

**Comment by forstisabella** [comment]
hey @tcgilbert - can you take a look at this PR when you get a chance?

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+<!-- This template is meant for Actions-based destinations that do not have an existing Classic or non-Actions-based version. For Actions Destinations that are a new version of a classic destination, see the doc-template-update.md template. -->
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/hyperengage/index.md:8]:
```suggestion
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+
+{% include content/plan-grid.md name="actions" %}
+
+<!-- Include a brief description of the destination here, along with a link to your website. -->
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/hyperengage/index.md:12]:
```suggestion
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 17]
```diff
@@ -0,0 +1,45 @@
...
+<!-- In the section below, explain the value of this actions-based destination. If you don't have a classic version of the destination, remove this section. -->
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/hyperengage/index.md:13]:
```suggestion
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 18]
```diff
@@ -0,0 +1,45 @@
...
+## Benefits of Hyperengage (Actions)
+
+Hyperengage (Actions) offers several advantages:
+
+- **Seamless Data Mapping**: Hyperengage streamlines the process of syncing your user and account data. When you link your data with Segment, Hyperengage automatically processes Segment's Identify, Track, and Group calls, eliminating the need for manual API integrations.
+- **Pre-configured Mappings**: The Hyperengage (Actions) Destination has prebuilt mappings tailored for Hyperengage with all the necessary parameters, reducing the need for manual customization.
+- **Direct Data Transfer**: Data moves straight from Segment to Hyperengage, eliminating the need for third-party intermediaries.
+- **Event Tracking and User Identification**: With Segment's Actions-based destination, you can keep track of events and identify users and organizations in Hyperengage.
+
+<!-- The section below describes how to activate and set up the destination. Include any additional configuration steps not mentioned. For example, acquiring an API key from your system and any setup steps needed to establish a connection to the destination. -->
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/hyperengage/index.md:25]:
```suggestion
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 19]
```diff
@@ -0,0 +1,45 @@
...
+
+1. Navigate to the Segment web app and select **Catalog**, then pick **Destinations**.
+2. In the Destinations Catalog, search for "Hyperengage" and select the Hyperengage (Actions) destination.
+3. Click **Add destination**.
+4. Choose an existing Source to connect to Hyperengage (Actions) and click **Next**.
+5. Enter a name for your destination and click **Create destination**.
+6. Open the [Hyperengage App](https://hyperengage.io/){:target="_blank"}, proceed to **Integration Settings**, and copy the API Key and Workspace Identifier.
+7. Open the Segment app, navigate to your Hyperengage (Actions) destination, and paste the API Key and Workspace Identifier into the destination's settings page.
+
+<!-- The line below will render a table of connection settings (if they exist), Pre-built Mappings, and potential actions. -->
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/hyperengage/index.md:37]:
```suggestion
```

File: src/connections/destinations/catalog/hyperengage/index.md [hunk 20]
```diff
@@ -0,0 +1,45 @@
...
+
+{% include components/actions-fields.html %}
+
+<!--
+Additional Context
+
+Include additional information that you think will be useful to the user here. For information that is specific to an individual mapping, please add that as a comment so that the Segment docs team can include it in the auto-generated content for that mapping.
+-->
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/hyperengage/index.md:45]:
```suggestion
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]
please remove template comments

**Comment by forstisabella** [comment]
hi @zhadier39 - I'm not able to directly edit this branch due to protections on your branch. I've duplicated the contents of this PR in #5606 and we'll use that one to merge the docs into the site!

**Pull request closed without merging**